bridge_adaptivity/docker-compose_local.yml

```diff
@@ -19,6 +19,9 @@ services:
       - .:/bridge_adaptivity
     ports:
       - "8008:8008"
+    extra_hosts:
```

Comment:
You could create your own local docker-compose.yml file and run local deployment which is based on it